deploy/deploy_common.sh

```diff
@@ -21,11 +21,6 @@ myloc=$(pwd)
 # Get full path of this script no matter where it's placed and invoked
 MY_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-if [[ ! -f $MY_PATH/playbooks/inventories/vars/.vault.yml ]]; then
```

Comment:
Don't make such change, env_setup is needed for local setup, you can touch create the needed file or add new condition to filter out situation.